pkg/cloud/services/networking/securitygroups_test.go

```diff
@@ -681,3 +681,63 @@ func TestService_ReconcileSecurityGroups(t *testing.T) {
 		})
 	}
 }
+
+// Unit Test for #2666
```

Comment:
I would also suggest removing this comment. We are simply testing `getSGControlPlaneAdditionalPorts`.

```suggestion
```

Reply:
(This will also fix the linter error)